.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - develop
+    tags:
+      - 'v*'
```

Comment:
There is a dedicated trigger that runs jobs when a release is created on a GH repository:

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Might be cleaner than triggering on tags.

Reply:
It depends on what is more ergonomic, which depends on the person :shrug:. I personally like to just push a tag and collect the automatically created release a few minutes later, without clicking over the web GUI. Ideally, the workflow could support both.